### PR TITLE
Redpanda console

### PR DESCRIPTION
**Pull request by KastTrifork**

### CODE REVIEW

**Comment by KastTrifork** [comment]
I'm struggling with this again error: couldn't find key client-Id in Secret redpanda-console-test/redpanda-oauth2-proxy.
I have checked secrets in k9s and the client-id are stored in redpanda-oauth2-proxy

**Comment by KastTrifork** [comment]
It works now, running on our cluster

File: charts/cheetah-redpanda-console/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+metadata:
+  name: {{ include "console.fullname" . }}
+  labels: {{- include "console.labels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
+  annotations: {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: 4180
```
Comment (pegtrifork) [inline charts/cheetah-redpanda-console/templates/service.yaml:29]: To reverting this file to the default from redpanda-console. And only set default values in "values.yaml" instead

File: charts/cheetah-redpanda-console/templates/ingress.yaml
Comment (pegtrifork) [inline charts/cheetah-redpanda-console/templates/ingress.yaml:1]: To reverting this file to the default from redpanda-console. And only set default values in "values.yaml" instead

File: charts/cheetah-redpanda-console/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+            periodSeconds: 5
+          securityContext:
+            runAsNonRoot: true
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop:
+                - ALL
+          ports:
+            - containerPort: 4180
```
Comment (pegtrifork) [inline charts/cheetah-redpanda-console/templates/deployment.yaml:321]: Should preferably also be taken from values, and should just match https://github.com/trifork/cheetah-charts/pull/216/files#diff-7f4165ef21ea4ac14fc7bc4e70899250ef7adeddae686abe79631a21aaa40e8eR29

File: charts/cheetah-redpanda-console/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,323 @@
+{
```
Comment (rkthtrifork) [inline charts/cheetah-redpanda-console/values.schema.json:1]: add these values

```yaml
        - name: oauth2-proxy
          image: quay.io/oauth2-proxy/oauth2-proxy:v7.5.1
          imagePullPolicy: IfNotPresent
          args:
            - --oidc-issuer-url={{.Values.oauth2Proxy.issuerUrl}}
            - --upstream=http://127.0.0.1:8080
            - --reverse-proxy=true
            - --email-domain={{.Values.oauth2Proxy.emailDomain}}
            - --cookie-secure=true
            - --cookie-name={{.Values.oauth2Proxy.cookieName}}
            - --provider=oidc
            - --http-address=0.0.0.0:{{.Values.service.port}}
            - --exclude-logging-path="/ping"
          readinessProbe:
            httpGet:
              path: /ping
              port: http-proxy
            initialDelaySeconds: 5
            periodSeconds: 5
          livenessProbe:
            httpGet:
              path: /ping
              port: http-proxy
            initialDelaySeconds: 5
            periodSeconds: 5
          securityContext:
            runAsNonRoot: true
            allowPrivilegeEscalation: false
            readOnlyRootFilesystem: true
            capabilities:
              drop:
                - ALL
          ports:
            - containerPort: {{ .Values.service.targetPort }}
              protocol: TCP
              name: http-proxy
          env:
        {{- with .Values.oauth2Proxy.env }}
        {{- toYaml . | nindent 12 }}
```

**Review by rkthtrifork — CHANGES_REQUESTED** [review]
see comments and consider adding istio VirtualService for ingress

File: charts/cheetah-redpanda-console/templates/NOTES.txt [hunk 4]
```diff
@@ -14,9 +14,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-This is helm chart is an extention of the console chart from the official helm chart repository.
+This is helm chart is an extension of the console chart from the official helm chart repository.
```
Comment (rkthtrifork) [inline charts/cheetah-redpanda-console/templates/NOTES.txt:17]:
```suggestion
This helm chart is an extension of the console chart from the official helm chart repository.
```

**Review by rkthtrifork — CHANGES_REQUESTED** [review]
typo

**Pull request merged**